tests/initial-blank-stories.jsx

```diff
@@ -0,0 +1,14 @@
+/*
+ * This file exists to place a buffer between
+ * the Express server loading and image snapshots
+ * running.
```

Comment:
Why?

Reply:
I'm thinking we should maybe change this to a "Welcome to DSR" heading centered page which would be kind of nice as the first story. What do you think?